src/Action/DashboardAction.php

```diff
@@ -16,11 +16,11 @@
 use Sonata\AdminBundle\Admin\BreadcrumbsBuilderInterface;
 use Sonata\AdminBundle\Admin\Pool;
 use Sonata\AdminBundle\Templating\TemplateRegistryInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-final class DashboardAction extends Controller
+final class DashboardAction extends AbstractController
 {
```

Comment:
IMHO we should not refer to any controller, we should register controller as services